ngraph/frontend/onnx_import/src/op/resize.cpp

```diff
@@ -292,6 +294,9 @@ namespace ngraph
                             data, sizes, scales, attrs)};
                     }
 
+                    attrs.shape_calculation_mode =
+                        default_opset::Interpolate::ShapeCalcMode::scales;
```

Comment:
Why these changes are in the PR which should modify a Constant operator?
Is it intentional?

Reply:
Yes, the whole PR enables a couple of models which are affected by both changes.